File: page-objects/java-fluentlenium/initial/src/main/java/pages/SearchResultsPage.java
package pages;

import org.fluentlenium.core.FluentPage;
import org.openqa.selenium.By;

public class SearchResultsPage extends FluentPage {

    public void clickOnFirstProduct() {
        find(By.cssSelector("z-grid[class='z-nvg-cognac_articles'] > z-grid-item:first-child")).click();
    }

}

File: page-objects/js-webdriverio/initial/pageobjects/searchResults.page.js
// searchResults.page.js
var Page = require('./page');

var SearchResultsPage = Object.create(Page, {
    /**
     * define elements
     */
    firstElement: { get: function () { return browser.element("z-grid[class='z-nvg-cognac_articles'] > z-grid-item:first-child"); } },

    clickOnFirstElement: { value: function() {
            this.firstElement.click();
        } }
});

module.exports = SearchResultsPage;